resources/js/pages/demos/alert/with-actions.tsx

```
@@ -0,0 +1,22 @@
+import { AlertCircleIcon } from '@hugeicons/core-free-icons'
```

Biome (2.4.4)

[error] This import is unused.

(lint/correctness/noUnusedImports)